### ISSUE TITLE
When to release the codes

### ISSUE THREAD

**Issue by jinx2018** [hxixixh/mix-and-localize#1]
Hi! Could you please release the source codes since the CVPR2022 has finished?

**Comment by hxixixh** [comment]
Hi @jinx2018, thanks for your concerns about our work. We're planning to release the code in the next week or two

**Comment by girmaji08** [comment]
Hi @hxixixh  Could you please share the code. It will be really helpful for my research. Thanks.

**Comment by hxixixh** [comment]
@jinx2018 @girmaji08 
Thanks for your interest! Sorry that I'll delay the code release until later next week. Please check back later.

**Comment by guglielmocamporese** [comment]
Hi,

thanks for your nice paper and for being accepted at CVPR 2022, I attended your posted session in New Orleans! 
I'm trying to re-implement your model on my own, but the loss is converging super-fast, and the model is not learning anything useful. 

This is the training loss on `MUSIC-synthetic`:
<img width="1124" alt="Screenshot 2022-09-16 at 18 25 24" src="https://user-images.githubusercontent.com/31989563/190685702-e974c40e-90e6-4e7b-a7e4-e73e019a036c.png">

I tried to apply data-augmentation (audio shift, as reported in the paper) for the computation of the matrix $A_{IS'}$, but still the issue persists. I'm using the same hyper-parameters as you reported in the paper, and I implemented a $128$-dimensional `resnet18` modifying the `torchvision.models.resnet` model (specifically I set `stride=1` in the`conv1` module in order to obtain a feature map of $128 \times 14 \times 28$ - here I'm stiching the input images horizontally..).

Hope to see your code soon, or some insights about the data-augmentation you used during training!

Best,

Guglielmo

**Comment by IFICL** [comment]
@girmaji08 Hi, thank you for your interest in our project. From your results, the model seems to learn a shortcut.
1. For data augmentation, you can try with a larger shift (it's okay if there is no overlap between them since it's single image setup)
2. For the dataset, I think we didn't train on `Music-Synthetic` dataset while we test on them. During the training, images are not stitched together. Every single image is encoded as 128 x 14 x 14 where we set the `stride=1` in the last of the residual layer. 

Sorry for the delay in code release, we are working on some other deadlines. Hopefully, we will release our code soon.

**Comment by hxixixh** [comment]
Hi @jinx2018 @girmaji08 @guglielmocamporese 
I'm sorry for the delayed update. I've released the code and please let me know if you have further questions : )

**Comment by girmaji08** [comment]
@hxixixh . Thanks a lot for the code. Do you have the pretrained model on VoxCeleb2 or preprocessing code for VoxCeleb2 dataset.?? That would really help my work. 
Thanks,
Rohit

**Comment by hxixixh** [comment]
@girmaji08 Hi Rohit, sure, I'll try to release the pretrained model recently, hopefully within this week

**Comment by girmaji08** [comment]
@hxixixh . Hi . Sorry to bother you again. Thanks a lot for the code for MUSIC Dataset. Do you have the pretrained model on VoxCeleb2 or preprocessing code for VoxCeleb2 dataset.?? That would really help my work.
Thanks,
Rohit

**Comment by hxixixh** [comment]
@girmaji08 Hi, I'm very sorry for the delay! Please refer to README to download the pretrained model on VoxCeleb2
